screens/authentication/login/login.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Cart</title>
+    <link
+      rel="shortcut icon"
+      href="../../../assets/logo.png"
```

Comment:
if possible use absolute paths

Reply:
Ok, got your point, Thanks for the feedback.